modules/gapi/CMakeLists.txt

```diff
@@ -156,6 +156,10 @@ set(gapi_srcs
     src/backends/ie/giebackend.cpp
     src/backends/ie/giebackend/giewrapper.cpp
 
+    # OV Backend. FIXME: should be included by CMake
+    # if and only if OV support is enabled
```

Comment:
The file already includes `HAVE_INF_ENGINE` so do we need an extra check here?

Reply:
This comment is relevant for `IE` and for `OV` nothing changed. We still include this header regardless `OpenVINO` exists or not.